### ISSUE TITLE
when will be the 2024 version released?

### ISSUE THREAD

**Issue by zeasa** [areuther/ai-accelerators#1]

**Comment by areuther** [comment]
Hello Zeasa, Thank you for your interest in the survey. I am preparing a 2024 version of the survey for IEEE HPEC (http://www.ieee-hpec.org/) submission. The extended deadline for that conference is July 14, 2024 this year. Are you more interested in the data, the paper, or the slides? This year, I may push to get the data posted in late July. I post the preprint paper on ArXiv.org after paper acceptances are announced (August 27 this year). And slides are usually available a few weeks after the conference, usually in mid-October. --Albert

**Comment by StefanHri** [comment]
Hi, are there any updates regarding the 2024 version? Great work by the way! I really enjoyed reading the papers.

**Comment by areuther** [comment]
Thank you, Stefan. I am still working on the 2024 version. In May and June, when I usually compile and write each year's edition, I found that I did not have enough new accelerators to justify a new version. I do still plan to post an updated dataset because I want to include the Nvidia B100 and the IBM NorthPole. My goal is to post that before the end of the year.

**Comment by AI-LJ** [comment]
+ AMD325X, TPUV5, Horizon-journey6 and so on......

**Comment by mandulaj** [comment]
Any update on 2025?

**Comment by areuther** [comment]
> Any update on 2025?

Indeed. I have collected quite a number of new accelerators' data and am writing the paper this week. Submissions are due on Monday. I should have an updated CSV posted to this site within a few weeks.

**Comment by mandulaj** [comment]
That's amazing! It would be really cool to keep this up-to date in a repository like yours so that it can be updated in the future easily. There is even a nice python visualization script in https://github.com/chang55245/ai-accelerators-viz/network which could easily be transformed into a static website hosted on GitHub that directly renders the most up to date data. Let me know if you would be interested in collaborating on this.

**Comment by mandulaj** [comment]
Here, did a quick preview version of it: https://mandulaj.github.io/ai-accelerators-viz/
